web/js/duplicates.js

```diff
@@ -190,6 +200,10 @@
         if ($('.js-responsibility-message:visible').length) {
             return false;
         }
+        // On mobile only show once per category
+        if ($('html').hasClass('mobile') && dismissed) {
```

Comment:
This means that on mobile, if you pick a point, get some dupes, go back up to the map and move the point to somewhere entirely different, you do not look up any duplicates around that new location (or rather they have been looked up, but this then stops them being shown). I understand the reasoning when there are assets, so perhaps this is acceptable, I'm not sure if there's a way that can accommodate it all (could only do this if assets are being shown I guess...).

Reply:
Yes, it's tricky isn't it. I think the prioritisation of assets over duplicates is the right choice in this case. Given the assets have fixed locations, perhaps the duplicates code should only suggest duplicates that are literally on the same asset? A future improvement in any case.